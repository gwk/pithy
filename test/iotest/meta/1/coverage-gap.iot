{
  'compile': [
    'touch .project-root',
  ],
  'cmd': 'iotest -no-times -coverage test-meta/1/$NAME.iot',
  'skip': True,
  'err_mode': 'ignore', # allow the coverage tool to print debug info without spoiling this test.
}
