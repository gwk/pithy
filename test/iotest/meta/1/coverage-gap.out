test-meta/1/coverage-gap

################################################################
TESTS FOUND: 1; SKIPPED: 0; PASSED: 1.

# coven -coalesce _build/test-meta/1/coverage-gap/coverage-gap._.coven
----------------
Coverage Report:

__main__: _build/test-meta/1/coverage-gap/coverage-gap.py:
   1   #!/usr/bin/env python3
   2
   3   def f():
   4 !   return None

__main__: _build/test-meta/1/coverage-gap/coverage-gap.py: 2 traceable; 1 covered; 0 ignored; 0 ignored but covered; 1 not covered.
